packages/web/server/queries/complex/assets/index.ts

```diff
@@ -33,11 +34,17 @@ const searchableAssetListAssetKeys = ["symbol", "base", "name", "display"];
 export async function getAsset({
   assetList = AssetLists,
   anyDenom,
+  ...params
 }: {
   assetList?: AssetList[];
   anyDenom: string;
-}): Promise<Asset | undefined> {
-  const assets = await getAssets({ assetList, findMinDenomOrSymbol: anyDenom });
+} & AssetFilter): Promise<Asset | undefined> {
+  const assets = await getAssets({
+    assetList,
+    findMinDenomOrSymbol: anyDenom,
+    includeUnlisted: true,
```

Comment:
Do we want this hardcoded here?

Reply:
I believe so, however, these params can be overriden if needed with the `...params`. For detailed queries, unlisted assets are included by default since these queries are like searches and are only used when a specific asset is selected.